tests/test_invessel_build.py

```diff
@@ -96,6 +96,7 @@ def test_ivb_basics(invessel_build):
     assert invessel_build.repeat == repeat_exp
     assert invessel_build.num_ribs == num_ribs_exp
     assert invessel_build.num_rib_pts == num_rib_pts_exp
+    assert len(invessel_build.Surfaces['component'].get_loci()) == num_rib_pts_exp
```

Comment:
I think we need to call `invessel_build.calculate_loci()` before we can `get` them

Reply:
Or this new test should come as part of the next test where we already call `calculate_loci()`

Reply:
sweet, I moved it there and then updated the method a bit as I realized as written here this test actually wouldn't have caught this issue.